docs/content/tutorial/saving-your-data.mdx

```diff
@@ -0,0 +1,179 @@
+    "base_dir": "data", # Path is built relative to where `dagster dev` is run
+})
+```
+
+Next, update your `Definitions` (shown below) to attach the I/O manager to your code location by defining it as a **resource**.
+
+```python
+defs = Definitions(
+    assets=hackernews_assets,
+    schedules=[hackernews_schedule],
+    resources={
+        "io_manager": io_manager,
+    }
+)
+```
+
+<Note>
+  <strong>About resources</strong>
+  <br />
+  I/O managers are a special type of resource, a Dagster concept for{" "}
+  <strong>connecting</strong> to external services. I/O managers are resources
+  made specifically to control reading and writing to external services. Refer
+  to the <a href="/concepts/resources">Resources documentation</a> for more
+  information.
+</Note>
+
+To see the effect of this change, reload your code location in the Dagster UI and materialize your assets. When you click on an asset, you’ll see the `path` metadata pointing to a local storage directory.
+
+Look at your Dagster project in your file system (via Finder or Windows Explorer) and you’ll find a new directory called `data` that contains your materialized assets.
+
+This is a short example of how powerful I/O managers are. In a few lines of code, assets previously stored in ephemeral storage can now be saved **anywhere**. With I/O managers, you can save your assets to places like AWS S3, Google Cloud Storage, or Azure Blob Storage.
+
+In addition to file storage systems, I/O managers can connect to any service from which data can be read or written, such as databases.
+
+---
+
+## Step 2: Writing to databases
+
+In data engineering, a common asset is tabular data. These are often seen as database tables or DataFrames in memory. I/O managers translate between tables and in-memory data formats.
+
+Dagster includes out-of-the-box IO managers for common databases, such as DuckDB. When a DataFrame is returned in an asset’s function, it is translated and written into the database.
+
+Let’s modify our pipeline to store our `topstories` DataFrame as a table in DuckDB. We’ll use Dagster’s out-of-the-box I/O manager for DuckDB, an easy-to-set-up database that doesn't require any long-running processes.
+
+### Setting up I/O managers
+
+Adding an I/O manager for a database is similar to connecting it to a file storage system. The biggest difference is you must specify what **type** of DataFrame to use when the data is loaded into Dagster’s memory. In this tutorial, you’ll use DuckDB to store the data and Pandas DataFrames to work with the data.
+
+Update your `__init__.py` to reflect the changes below:
+
+```python
+from dagster_duckdb import build_duckdb_io_manager
+from dagster_duckdb_pandas import DuckDBPandasTypeHandler
+# Add the imports to the top
+# These imports let you define how Dagster communicates with DuckDB
+
+# Insert this section anywhere above your `defs = Definitions(...)`
+duckdb_io_manager = build_duckdb_io_manager([DuckDBPandasTypeHandler()]).configured({
```

Comment:
I think it would be simpler to just use `duckdb_pandas_io_manager`, to avoid needing to go into the TypeHandler business.